internal/commands/database/create.go

```diff
@@ -0,0 +1,214 @@
+	}
+	return nil
+}
+
+func processProperties(in []string) (request.ManagedDatabasePropertiesRequest, error) {
+	resp := request.ManagedDatabasePropertiesRequest{}
+	for _, prop := range in {
+		parts := strings.SplitN(prop, "=", 2)
+		if len(parts) != 2 {
+			return resp, fmt.Errorf("invalid property format: %s, expected key=value", prop)
+		}
+
+		key := upcloud.ManagedDatabasePropertyKey(parts[0])
+		value := parts[1]
+
+		// remove quotes from the start and end of the value if they exist
+		if (strings.HasPrefix(value, "\"") && strings.HasSuffix(value, "\"")) ||
+			(strings.HasPrefix(value, "'") && strings.HasSuffix(value, "'")) {
+			value = value[1 : len(value)-1]
+		}
+
+		var parsedValue interface{}
+		if err := json.Unmarshal([]byte(value), &parsedValue); err != nil {
+			resp.Set(key, value) // set as plain string if parsing fails
+		} else {
+			resp.Set(key, parsedValue)
+		}
+	}
+	return resp, nil
+}
+
+func processNetworks(in []string) ([]upcloud.ManagedDatabaseNetwork, error) {
+	var networks []upcloud.ManagedDatabaseNetwork
+	for _, netStr := range in {
+		network := upcloud.ManagedDatabaseNetwork{}
+		pairs := strings.Split(netStr, ",")
+
+		for _, pair := range pairs {
+			parts := strings.SplitN(pair, "=", 2)
+			if len(parts) != 2 {
+				return nil, fmt.Errorf("invalid network format: %s, expected key=value", pair)
+			}
+
+			key := strings.TrimSpace(parts[0])
+			value := strings.TrimSpace(parts[1])
+
+			switch key {
+			case "family":
+				network.Family = value
+			case "name":
+				network.Name = value
+			case "type":
+				network.Type = value
+			case "uuid":
+				network.UUID = &value
+			default:
+				return nil, fmt.Errorf("unknown network parameter: %s", key)
+			}
+		}
+		networks = append(networks, network)
+	}
+	return networks, nil
+}
+
+// InitCommand implements commands.InitializeCommand
+func (s *createCommand) InitCommand() {
+	flags := &pflag.FlagSet{}
+	s.params = createParams{CreateManagedDatabaseRequest: request.CreateManagedDatabaseRequest{}}
+	def := defaultCreateParams
+	flags.StringVar(&s.params.HostNamePrefix, "host-name-prefix", def.HostNamePrefix, "A host name prefix for the database")
+	flags.StringVar(&s.params.Title, "title", def.Title, "A short, informational description.")
+	flags.StringVar(&s.params.Plan, "plan", def.Plan, "Plan to use for the database. Run `upctl database plans [database type]` to list all available plans.")
+	flags.StringVar(&s.params.Zone, "zone", def.Zone, namedargs.ZoneDescription("database"))
+	flags.StringVar(&s.params.dbType, "type", string(def.Type), "Type of the database")
+	flags.StringVar(&s.params.Maintenance.DayOfWeek, "dow", def.Maintenance.DayOfWeek, "Full name of weekday in English, lower case(sunday) for automatic maintenance day of the week. Set randomly if not provided.")
+	flags.StringVar(&s.params.Maintenance.Time, "time", def.Maintenance.Time, "Database time in UTC of automatic maintenance HH:MM:SS. Set randomly if not provided.")
```

Comment:
and maybe these two could have `maintenance-` prefix to be more self explaining 🤔 

```suggestion
	flags.StringVar(&s.params.Maintenance.DayOfWeek, "maintenance-dow", def.Maintenance.DayOfWeek, "Full name of weekday in English, lower case(sunday) for automatic maintenance day of the week. Set randomly if not provided.")
	flags.StringVar(&s.params.Maintenance.Time, "maintenance-time", def.Maintenance.Time, "Database time in UTC of automatic maintenance HH:MM:SS. Set randomly if not provided.")
```